fabric/bin/lib/common_ledger.sh

```diff
@@ -7,19 +7,23 @@
 # - CONFIG_HOME is defined
 # - common_utils.sh is loaded
 # optional config overrides
+# - FABRIC_PATH
```

Comment:
you might want to reuse $FABRIC_PATCH from `config.mk`

Reply:
Not sure what you are referring to: this is a shell script which should be executable separate from make, so can't use config.mk.  It also defines currently imho sensible defaults consistent with config.mk and i mention it in the comment only as docu that you can override defaults ...